### ISSUE TITLE
How is this different from leknoppix/ckeditor5-fullscreen 

### ISSUE THREAD

**Issue by jameswilson** [pikulinpw/ckeditor5-fullscreen#2]
https://github.com/leknoppix/ckeditor5-fullscreen exists since 2020.

How is this different?

**Comment by syntaxsurge** [comment]
This has been published as npm package

**Comment by pikulinpw** [comment]
Hello, there was already an answer above, I would also add that my variant uses FullScreen API
Also within a month there will be an update that will improve it many times more.

**Comment by jameswilson** [comment]
Makes sense.  Thanks!  Looks like the maintainer of the other version admits some shortcomings here: 

https://github.com/leknoppix/ckeditor5-fullscreen/issues/4

What you sometimes see is a section on the main Readme for similar projects and how they're different, so I could recommend you could add such a section, if you're so inclined before you close this.
